keps/prod-readiness/sig-storage/625.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 625
+stable:
+  approver: "@msau42"
```

Comment:
Could you actually change this to wojtek-t since he started reviewing PRR for this?
Unfortunately, msau42 is not one of the approvals. 

you can find valid approvers listed [here](https://github.com/kubernetes/community/blob/master/sig-architecture/production-readiness.md#production-readiness-review-process)

Reply:
Sure, thanks for noticing!